managers/engineering_manager.md

```diff
@@ -27,28 +29,28 @@ Engineering Manager
 
 #### Meetings
 
-* Guide effectiveness of team meetings. Ensure agendas/purpose are clear & actions captured. 
+* Guide effectiveness of team meetings.
+* Ensure agendas/purpose are clear & actions captured.
 
 #### Determine team/team makeup & hiring needs
 
 * Ensure team has the people and skill sets needed to achieve their goals
-* Recruit for the team 
+* Recruit for the team
    * Source engineers
    * Motivate engineers to speak, do presentations, write blog posts, and contribute to open source projects (and do so oneself!)
-   * Participate in creating interview practice for Mariana Tek engineering
-   * Update interview loops & training engineers to use them for hiring 
+   * Update interview loops & training engineers to use them for hiring
    * Work with recruiting to ensure the loops run smoothly
    * Make the final hiring decision
-   * Create the engineer’s onboarding plan and work with TL and others to ensure engineers are successfully onboarded 
+   * Create the engineer’s onboarding plan and work with TL and others to ensure engineers are successfully onboarded
 
 #### Management
 
 * Ensure engineers understand how the work they are doing ladders up to OKRs & wider company goals
 * Ensure engineers have the skills needed to support their team
-* Ensure engineers are focussed on the sprint & stories at hand
+* Ensure engineers are focused on the sprint & stories at hand
 * Deal with team interpersonal issues
-* Deal with performance issues 
-* Coach engineers career paths and personal growth in a way that lines up with teams goals
+* Deal with performance issues
```

Comment:
I'm reading this as really talking about personnel performance - I wonder if this might be too general at present?

Specifically - I'm thinking about whether we might instead speak directly to the manager's tasks of: providing feedback to their direct reports on a regular and timely basis, ensuring that regular reviews are available for the team, etc.

I feel like grounding these section more might be good - I'm also happy to throw up another PR with some edits for exactly that so as not to block these wonderful changes!

Reply:
Agreed, this could be improved. I also love the idea of tag teaming, so if you want to put up another PR let's do it!

Reply:
Cool - on my list! Thanks @adsteel!